module_4/src/service/login.js

```diff
@@ -0,0 +1,31 @@
+import { LogService } from './index.js';
+import { JWT as jwtConfig } from '../../config/index.js';
+import jwt from 'jsonwebtoken';
+
+export class LoginService extends LogService {
```

Comment:
not sure if it's a good idea to inherit LogService this way.
Application logger - it is a utility which is just another dependency for your services, not a building block.